docs/source/reference/modding/font.rst

```diff
@@ -9,116 +9,35 @@ To our knowledge, the format is undocumented.
 
 OpenMW can load this format and convert it on the fly into something usable 
 (see font loader `source code <https://github.com/OpenMW/openmw/blob/master/components/fontloader/fontloader.cpp#L210>`_). 
-In OpenMW 0.32, an --export-fonts command line option was added to write the converted font 
+You can use an --export-fonts command line option was added to write the converted font
 (a PNG image and an XML file describing the position of each glyph in the image) to the current directory.
 
 TrueType fonts
 --------------
 
 Unlike vanilla Morrowind, OpenMW directly supports TrueType (``.ttf``) fonts.
 
-0.45.0+ way
------------
 This is the recommended way to install replacement fonts.
 
--	To replace the primary "Magic Cards" font:
+	1.	Download `TrueType fonts for OpenMW <https://www.nexusmods.com/morrowind/mods/46854>`_
+	2.	Place the ``Fonts`` folder from archive to the configuration folder. It is located as described in :doc:`paths`.
 
-	1.	Download `Pelagiad <https://isaskar.github.io/Pelagiad/>`_ by Isak Larborn (aka Isaskar).
-	2.	Create ``Fonts`` folder at the location of your ``openmw.cfg``.
-	3.	Copy ``openmw_font.xml`` and ``Pelagiad.ttf`` files into the folder.
-	4.	Either remove the ``MonoFont`` entry from the created ``openmw_font.xml`` or copy the ``DejaVuLGCSansMono.ttf`` to the ``Fonts`` folder (prefer latter option on Windows).
-	5.	If desired, you can now delete the original ``Magic_Cards.*`` files from your ``Data Files/Fonts`` directory.
-	6.	Pelagiad glyphs may appear to be pretty small, so open ``settings.cfg`` and adjust the following settings as you deem necessary::
+As a result, you should have the Fonts folder with ``openmw_font.xml`` and three ``.ttf`` files.
 
-			[GUI]
-			font size = 17
-			ttf resolution = 96
+If desired, you can now delete the ``Data Files/Fonts`` directory.
 
--	You can also replace the Daedric font:
+Also it is possible to adjust the text size and resolution::
 
-	1.	Download `Ayembedt <https://github.com/georgd/OpenMW-Fonts>`_ by Georg Duffner.
-	2.	Install ``OMWAyembedt.otf`` into the ``Fonts`` folder.
-	3.	Add the following lines into ``openmw_font.xml``::
-
-			<Resource type="ResourceTrueTypeFont" name="Daedric">
-				<Property key="Source" value="OMWAyembedt.otf"/>
-				<Property key="Antialias" value="false"/>
-				<Property key="TabWidth" value="8"/>
-				<Property key="OffsetHeight" value="0"/>
-				<Codes>
-					<Code range="32"/>
-					<Code range="65 90"/>
-					<Code range="97 122"/>
-				</Codes>
-			</Resource>
+			[GUI]
+			font size = 16
+			ttf resolution = 96
 
-	4.	This font is missing a few glyphs (mostly punctuation), but it has all the primary glyphs. If desired, you can now delete the original ``daedric.*`` files from your ``Data Files/Fonts`` directory.
+The ``font size`` can be from 12 to 20, the ``ttf resolution`` - from 48 to 960.
```

Comment:
``font size`` setting accepts clamped values in range from 12 to 20 while ``ttf resolution`` setting accepts values from 48 to 960.